lib/browser/navigation.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+var Url = require('url');
+
+var _ = require('lodash');
+var debug = require('debug')('testium-driver-sync:navigation');
+
+var makeUrlRegExp = require('./makeUrlRegExp');
+var waitFor = require('./wait');
+
+exports._forwarded = [
+  'refresh',
+  'getUrl',
```

Comment:
Reading the airbnb style guide now - some really good points.  What led you to start using it?